features/Extended Materials/Shaders/ExtendedMaterials/ExtendedMaterials.hlsli

```diff
@@ -14,14 +14,18 @@ struct DisplacementParams
 
 namespace ExtendedMaterials
 {
-	float AdjustDisplacement(float displacement, DisplacementParams params)
+	float ScaleDisplacement(float displacement, DisplacementParams params)
+	{
+		return (displacement - 0.5) * params.HeightScale;
+	}
+	float AdjustDisplacementNormalized(float displacement, DisplacementParams params)
```

Comment:
nitpick: can you add a spacing between all funcs

Reply:
This feels like a clang format thing.